stable/rethinkdb/values.yaml

```diff
@@ -1,7 +1,7 @@
 # Specs for the RethinkDB image
 image:
   name: codylundquist/helm-rethinkdb-cluster
-  tag: 0.1.0
+  tag: 0.2.0
```

Comment:
I've already built the new `codylundquist/helm-rethinkdb-cluster` docker image and tagged it with `0.2.0`.